### PR TITLE
corrected FC topics, snippets, and links

### PR DESCRIPTION
**Pull request by HUSSAR-mtrela**

### CODE REVIEW

File: en/components/financialchart.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
- 
-The financial chart component analyzes and selects data columns - Date/Time column to use for x-axis and Open, High, Low, Close, Volume columns or the first 5 numeric columns for y-axis. Beyond this, the user can specify the chart type to either Bar, Candle, Column, and Line. With its five main visual elements, a toolbar, a prices view, a volume view, an indicators view, and a navigation view, data has become more perceptible.  
+The **igx-financial-chart** is a charting component that makes it easy to visualize financial data by using a simple and intuitive API. Once the user binds the data, the chart offers multiple ways in which the data can then be visualized and interpreted. There are several display modes for price and volume and a host of financial indicators. The chart also makes use of label formatting to give context to the data.
```
Comment (pamelabrasil) [inline en/components/financialchart.md]: please revert this back to Financial Chart - this paragraph came from marketing, its SEO specific

Comment (HUSSAR-mtrela) [reply en/components/financialchart.md]: I changed it back.

File: en/components/financialchart.md [hunk 2]
```diff
@@ -62,11 +78,10 @@ export class AppModule {}
 ### Usage
 Now that the financial chart module is imported, next step is to bind it to data.
 
-In order to create an igxFinancialChart control, you must first have data to bind it to. The following code snippet demonstrates how to create a simple data source. 
+In order to create an igxFinancialChart control, you must first have data to bind it to. The following code snippet demonstrates how to create a simple data source.
```
Comment (gmurray81) [inline en/components/financialchart.md]: This is still wrong here.

File: en/components/financialchart_binding_to_multiple_data.md [hunk 3]
```diff
@@ -1,11 +1,11 @@
...
-_keywords: Ignite UI for Angular, Angular, Native Angular Components Suite, Native Angular Controls, Native Angular Components, Native Angular Components Library, Angular Chart, Angular Chart Control, Angular Chart Example, Angular Grid Component, Angular Chart Component, Angular Financial Chart 
+_description: The Ignite UI for Angular Financial Chart component is easily configured to display financial data using a simple and intuitive API, as once the user binds the data, the chart offers multiple ways in which the data can then be visualized and interpreted.
+_keywords: Ignite UI for Angular, Angular, Native Angular Components Suite, Native Angular Controls, Native Angular Components, Native Angular Components Library, Angular Chart, Angular Chart Control, Angular Chart Example, Angular Grid Component, Angular Chart Component, Angular Financial Chart
 ---
-## Binding Financial Chart to Multiple Data Sources 
+## Binding Financial Chart to Multiple Data Sources
 
-The igxFinancialChart can bind easily to multiple data sources, as shown in the following demo. 
+The **igx-financial-chart** control can bind easily to multiple data sources, as shown in the following demo.
```
Comment (gmurray81) [inline en/components/financialchart_binding_to_multiple_data.md]: use backticks please

File: en/components/financialchart_custom_tooltips.md [hunk 4]
```diff
@@ -1,11 +1,11 @@
...
+_keywords: Ignite UI for Angular, Angular, Native Angular Components Suite, Native Angular Controls, Native Angular Components, Native Angular Components Library, Angular Chart, Angular Chart Control, Angular Chart Example, Angular Grid Component, Angular Chart Component, Angular Financial Chart
 ---
-## Custom Tooltips 
+## Custom Tooltips
 
-The igxFinancialChart provides default tooltips for each type of series. The default tooltips displays all the information relevant to the particular series item for example series title, data values, axis values. They are styled to match the series' style. If default tooltips are not sufficient, custom tooltips can be configured to customize the tooltip content and look and feel.
+The **igx-financial-chart** control provides default tooltips for each type of series. The default tooltips displays all the information relevant to the particular series item for example series title, data values, axis values. They are styled to match the series' style. If default tooltips are not sufficient, custom tooltips can be configured to customize the tooltip content and look and feel.
```
Comment (gmurray81) [inline en/components/financialchart_custom_tooltips.md]: backticks please

File: en/components/financialchart_explore_chart_types.md [hunk 5]
```diff
@@ -1,21 +1,21 @@
...
+_description: The Ignite UI for Angular Financial Chart component is easily configured to display financial data using a simple and intuitive API, as once the user binds the data, the chart offers multiple ways in which the data can then be visualized and interpreted.
+_keywords: Ignite UI for Angular, Angular, Native Angular Components Suite, Native Angular Controls, Native Angular Components, Native Angular Components Library, Angular Chart, Angular Chart Control, Angular Chart Example, Angular Grid Component, Angular Chart Component, Angular Financial Chart
 ---
-## Financial Chart Axis 
+## Financial Chart Axis
 
-In the Financial Chart control, an axis provides base properties for specifying appearance of axis main lines, tickmarks, titles, and axis labels. 
+In the **igx-financial-chart** control, an axis provides base properties for specifying appearance of axis main lines, tickmarks, titles, and axis labels.
```
Comment (gmurray81) [inline en/components/financialchart_explore_chart_types.md]: backticks

File: en/components/financialchart_high_volume_data.md [hunk 6]
```diff
@@ -1,10 +1,11 @@
...
-_keywords: Ignite UI for Angular, Angular, Native Angular Components Suite, Native Angular Controls, Native Angular Components, Native Angular Components Library, Angular Chart, Angular Chart Control, Angular Chart Example, Angular Grid Component, Angular Chart Component, Angular Financial Chart 
+_description: The Ignite UI for Angular Financial Chart component is easily configured to display financial data using a simple and intuitive API, as once the user binds the data, the chart offers multiple ways in which the data can then be visualized and interpreted.
+_keywords: Ignite UI for Angular, Angular, Native Angular Components Suite, Native Angular Controls, Native Angular Components, Native Angular Components Library, Angular Chart, Angular Chart Control, Angular Chart Example, Angular Grid Component, Angular Chart Component, Angular Financial Chart
 ---
 ## High Volume Data
-The igxFinancialChart is capable of handling high volumes of data as shown in the following demo. 
+
+The **igx-financial-chart** control is capable of handling high volumes of data. The following demo binds 20 years of data, with 1 hour intervals and displays the stock prices using line chart type.
```
Comment (gmurray81) [inline en/components/financialchart_high_volume_data.md]: backticks

File: en/components/financialchart_panes.md [hunk 7]
```diff
@@ -1,16 +1,16 @@
...
-_keywords: Ignite UI for Angular, Angular, Native Angular Components Suite, Native Angular Controls, Native Angular Components, Native Angular Components Library, Angular Chart, Angular Chart Control, Angular Chart Example, Angular Grid Component, Angular Chart Component, Angular Financial Chart 
+_description: The Ignite UI for Angular Financial Chart component is easily configured to display financial data using a simple and intuitive API, as once the user binds the data, the chart offers multiple ways in which the data can then be visualized and interpreted.
+_keywords: Ignite UI for Angular, Angular, Native Angular Components Suite, Native Angular Controls, Native Angular Components, Native Angular Components Library, Angular Chart, Angular Chart Control, Angular Chart Example, Angular Grid Component, Angular Chart Component, Angular Financial Chart
 ---
-## Financial Chart Panes 
+## Financial Chart Panes
 
-The Financial Chart has multiple panes to display financial data. 
+The **igx-financial-chart** control has multiple panes to display financial data.
```
Comment (gmurray81) [inline en/components/financialchart_panes.md]: backticks

File: en/components/financialchart_performance.md [hunk 8]
```diff
@@ -1,11 +1,11 @@
...
+_description: The Ignite UI for Angular Financial Chart component is easily configured to display financial data using a simple and intuitive API, as once the user binds the data, the chart offers multiple ways in which the data can then be visualized and interpreted.
+_keywords: Ignite UI for Angular, Angular, Native Angular Components Suite, Native Angular Controls, Native Angular Components, Native Angular Components Library, Angular Chart, Angular Chart Control, Angular Chart Example, Angular Grid Component, Angular Chart Component, Angular Financial Chart
 ---
-## Performance 
+## Performance
 
-The igxFinancialChart is capable of handling high volumes of data. The following demo binds 20 years of data, with 1 day intervals and displays the stock prices using candle sticks.
+The **igx-financial-chart** control has very good performance of rendering a lot of data points. The following demo binds 20 years of data, with 1 day intervals and displays the stock prices using candle sticks.
```
Comment (gmurray81) [inline en/components/financialchart_performance.md]: backticks

**Pull request merged**